src/v2/Apps/NewForYou/newForYouRoutes.tsx

```diff
@@ -1,14 +1,29 @@
-import { NewForYouApp } from "./NewForYouApp"
 import { AppRouteConfig } from "v2/System/Router/Route"
+import { graphql } from "relay-runtime"
+import loadable from "@loadable/component"
+
+const NewForYouApp = loadable(() => import("./NewForYouApp"), {
+  resolveComponent: component => component.NewForYouAppFragmentContainer,
+})
 
 export const newForYouRoutes: AppRouteConfig[] = [
   {
     path: "/new-for-you",
     getComponent: () => NewForYouApp,
     onServerSideRender: ({ req, res }) => {
-      if (!req.user) {
+      if (!res.locals.sd.FEATURE_FLAGS.new_for_you || !req.user) {
```

Comment:
Nice 👍

Reply:
Yeah, I think someone suggested this to me at some point. 🤔 😉